### PR TITLE
Nexus 4539 ITs testing autoblock

### PR DESCRIPTION
**Pull request by velo**
Created 3 ITs to check autoblock.  
Also changed API to make autoblock configurable using System.properties

### CODE REVIEW

**Comment by velo** [comment]
Nice, it already include the discussion we started during commit to pull request! =D

File: nexus/nexus-test-harness/nexus-test-harness-launcher/src/main/java/org/sonatype/nexus/test/booter/Jetty7NexusBooter.java [hunk 1]
```diff
@@ -156,6 +156,9 @@ public class Jetty7NexusBooter
         // The proper fix is to make all Indexer related ITs behave "properly" (with some heuristics?), and have some
         // sort of "try-wait-try-failAfterSomeRetries" the search operation itself.
         System.setProperty( "mavenIndexerBlockingCommits", Boolean.TRUE.toString() );
+        
+        // NEXUS-4539 - to get test faster reduced autoblock check from 5 minutes to 30 seconds
+        System.setProperty( "plexus.autoblock.remote.status.retain.time", String.valueOf( 30*1000 ) );
```
Comment (cstamas) [inline nexus/nexus-test-harness/nexus-test-harness-launcher/src/main/java/org/sonatype/nexus/test/booter/Jetty7NexusBooter.java]: Why doing this _IT wise_? This way you affect nexus behavior in all ITs, not only in this one.

This should be moved to some of the NEXUS-4539 related IT...

Comment (velo) [inline nexus/nexus-test-harness/nexus-test-harness-launcher/src/main/java/org/sonatype/nexus/test/booter/Jetty7NexusBooter.java]: ok, done!

**Comment by cstamas** [comment]
Not a blocker, but just created https://issues.sonatype.org/browse/NEXUS-4620

File: nexus/nexus-test-harness/nexus-test-harness-launcher/src/main/java/org/sonatype/nexus/test/booter/Jetty7NexusBooter.java (hunk 1 above)
Comment (nabcos) [inline nexus/nexus-test-harness/nexus-test-harness-launcher/src/main/java/org/sonatype/nexus/test/booter/Jetty7NexusBooter.java]: We should add backup/restore of the system properties in @BeforeClass/@AfterClass for the AbstractNexusIntegrationTest.

**Comment by nabcos** [comment]
+1

**Pull request merged**